extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -382,6 +382,14 @@ export class AmpIframe extends AMP.BaseElement {
       return;
     }
 
+    if (height < 100) {
+      user.warn(TAG_,
+          'ignoring embed-size request because the resize height is ' +
+          'less than 100px',
+          this.element);
```

Comment:
we should also mention this in the .md file.
also should this be a warning or an error/assert?